src/js/themes/hpe.js

```diff
@@ -851,18 +851,33 @@ export const hpe = deepFreeze({
     },
     warning: {
       background: 'validation-warning',
```

Comment:
we also need to add:
```
global: {
   background: 'validation-warning',
}
```
and the equivalent for all the statuses. right now it's pulling the default from base.js which is a different color than our validation colors.

Reply:
thank you for clarifying I pushed up changes